app/src/main/java/me/echeung/moemoekyun/ui/activity/MainActivity.kt

```diff
@@ -137,28 +137,26 @@ class MainActivity : BaseActivity() {
 
     override fun initAppbar() {
         super.initAppbar()
-        supportActionBar!!.setDisplayHomeAsUpEnabled(false)
-        supportActionBar!!.setDisplayShowTitleEnabled(false)
+        supportActionBar?.apply {
+            setDisplayHomeAsUpEnabled(false)
+            setDisplayShowTitleEnabled(false)
+        }
 
         searchMenu = appbar?.findViewById(R.id.appbar_search_menu)
-        searchMenu!!.setOnMenuItemClickListener(object : ActionMenuView.OnMenuItemClickListener {
-            override fun onMenuItemClick(menuItem: MenuItem): Boolean {
-                return onOptionsItemSelected(menuItem)
-            }
-        })
+        searchMenu?.setOnMenuItemClickListener { menuItem -> onOptionsItemSelected(menuItem) }
```

Comment:
Could potentially just pass the reference to `onOptionsItemSelected` instead of having a lambda.